### ISSUE TITLE
ignore_module not working for remote module call

### ISSUE THREAD

**Issue by jaywithersbrewin** [terraform-linters/tflint#1760]
### Summary

Hi,

I'm trying to ignore not having a version defined on a remote module.
The module source is "app.terraform.io/rbcbrewindolphin/naming_standard/bd"

I have tried using .tflint.hcl and CLI but still getting:
Warning: module "naming_standard" should specify a version (terraform_module_version)

### Command

tflint --ignore-module="app.terraform.io/rbcbrewindolphin/naming_standard/bd"

### Terraform Configuration

```terraform
module "naming_standard" {
  source = "app.terraform.io/rbcbrewindolphin/naming_standard/bd"
}
```

### TFLint Configuration

```HCL
config {
    ignore_module = {
        "app.terraform.io/rbcbrewindolphin/naming_standard/bd" = true
    }
}
```

### Output

```Shell
Warning: module "naming_standard" should specify a version (terraform_module_version)

  on main.tf line 1:
   1: module "naming_standard" {

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.2.2/docs/rules/terraform_module_version.md
```

### TFLint Version

0.45.0

### Terraform Version

1.4.6

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows

**Comment by jaywithersbrewin** [comment]
Thanks, Ben.

Can you please provide an example as i'm confused.

Thanks!


[cid:2228d9d3-f2a9-48a3-8815-f53e0171c606]



________________________________

**Comment by jaywithersbrewin** [comment]
Worked out now, thanks anyway.


[cid:913e27ba-a05d-4790-8168-0cc55e1462f0]



________________________________
